### PR TITLE
Merge pull request #8304 from hannah-saurusrex/mobile-styles-update

### PR DESCRIPTION
**Pull request by hannah-saurusrex**
Issue: #4896 

## What I did
per issue #4896, I adjusted mobile styles so that bottom bar does not obscure content. 

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]

File: examples/official-storybook/manager-head.html [hunk 1]
```diff
@@ -2,4 +2,10 @@
   #manager-head-not-loaded {
     display: none;
   }
+  
+  @media (hover: none) {
+    div > div > div > ul {
```
Comment (shilman) [inline examples/official-storybook/manager-head.html]: Can we make this more specific, with a class or an id? This seems like a potentially dangerous selector

**Review by shilman — DISMISSED** [review]
Sorry for the slot turnaround on this PR -- thanks so much for putting it together! Any chance you can give it a small tweak per my comment?

File: examples/official-storybook/manager-head.html (hunk 1 above)
Comment (ndelangen) [reply examples/official-storybook/manager-head.html]: I agree

**Comment by ndelangen** [comment]
Something changed so that the `ul` this PR was targeting changed to another element-type.

I think I found an improved fix by setting `overflow: hidden` on the stories container.

**Pull request merged**